cli/console.ts

```diff
@@ -92,6 +92,22 @@ export function print(text: string) {
   writeStdOut(text);
 }
 
+export function formatStackTraceForQuietCompilation(compileError: dataform.ICompilationError): string {
```

Comment:
I have a few suggestions about this one:

1. Maybe let's move the function closer to `printCompiledGraphErrors` because it is used there?
2. We do not need `export` keyword here, it's internal function.
3. The formatting inside function seems broken.

Reply:
We use 2 spaces at the moment. LGTM, thanks!